end_to_end/eval_assert.py

```diff
@@ -20,13 +21,32 @@ def read(metrics_file, target):
   return avg
 
 
+def test_tflops(metrics_file, target, threshold):
+  """Asserts over tflops values"""
+  avg_tflops = read(metrics_file, target)
+  assert avg_tflops >= threshold
+
+
+def test_checkpointing(metrics_file, target):
+  """Asserts over loss values from loaded checkpoint"""
+  metrics_file_saved = 'saved_' + metrics_file
+  metrics_file_restored = 'restored_' + metrics_file
+
+  with open(metrics_file_saved, 'r', encoding='utf8') as saved,\
+    open(metrics_file_restored, 'r', encoding='utf8') as restored:
+    saved_loss = json.loads(saved.readlines()[-1])[target]
+    restored_loss = json.loads(restored.readlines()[0])[target]
```

Comment:
Why does the saved_loss grab index [-1] but the restored_loss grabs index [0]? Should both grab index [-1]?